bignum-serialization-kotlinx/build.gradle.kts

```diff
@@ -66,74 +69,80 @@ signing {
 }
 
 kotlin {
-
-    val hostOs = getHostOsName()
-    if (hostOs == primaryDevelopmentOs) {
-        jvm()
-        js(IR) {
+    jvm()
+    js {
+        compilations {
+            this.forEach {
+                it.compileTaskProvider.configure {
+                    kotlinOptions.sourceMap = true
+                    kotlinOptions.metaInfo = true
+                    if (it.name == "main") {
+                        kotlinOptions.main = "call"
+                    }
+                }
+            }
             nodejs()
-            browser()
-        }
-        wasmJs {
-            browser()
+            browser() {
+                testTask {
+                    useKarma {
+                        useChromeHeadless()
+                    }
+                }
+            }
         }
     }
-    if (hostOs == HostOs.LINUX) {
-        linuxX64()
-//    linuxArm32Hfp() Not supported by kotlinx serialization
-//    linuxArm64() Not supported by kotlinx serialization
-    }
+    linuxX64()
+    linuxArm64()
+    androidNativeX64()
+    androidNativeX86()
+    androidNativeArm32()
+    androidNativeArm64()
     iosX64()
     iosArm64()
     iosSimulatorArm64()
     macosX64()
     macosArm64()
-    tvos()
+    tvosArm64()
     tvosSimulatorArm64()
-    watchos()
+    tvosX64()
+    watchosArm32()
+    watchosArm64()
+    watchosDeviceArm64()
+    watchosX64()
     watchosSimulatorArm64()
-//    mingwX86() Not supported by kotlinx serialization
     mingwX64()
+    wasmJs {
+        browser()
```

Comment:
Not sure is right

Reply:
Me neither, I'll leave just the browser and see if anyone complains.